spanner/mutation_test.go

```diff
@@ -430,6 +432,139 @@ func TestStructToMutationParams(t *testing.T) {
 	}
 }
 
+func TestStructToMutationParams_Generated(t *testing.T) {
+	type Generated struct {
+		ID   int64
+		Name string `spanner:",generated"`
```

Comment:
Why is the preceding comma needed here? Why can't it just be `spanner:"generated"`?

Reply:
I am keeping the first value before comma to be parsed as alternate name,  and comma will be separator for future spanner tag values similar to how GORM has
https://gorm.io/docs/models.html#Fields-Tags